code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,356 @@
+	effect_desc = "Makes the hit creature highly flammable, or creates a plasma cloud."
+
+/obj/item/slimecross/destabilized/darkpurple/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.fire_stacks += 5
+		L.reagents.add_reagent("plasma", 10)
+	else
+		var/datum/reagents/R = new/datum/reagents(10)
+		R.add_reagent("plasma", 10)
+		var/datum/effect_system/smoke_spread/chem/smoke = new
+		smoke.set_up(R, 1, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/darkblue
+	colour = "dark blue"
+	effect_desc = "Freezes the hit creature in a block of solid ice."
+
+/obj/item/slimecross/destabilized/darkblue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.apply_status_effect(/datum/status_effect/freon/destabilized)
+
+/obj/item/slimecross/destabilized/silver
+	colour = "silver"
+	effect_desc = "Feeds the hit creature an unhealthy amount."
+
+/obj/item/slimecross/destabilized/silver/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.nutrition = NUTRITION_LEVEL_FAT + 1
+		to_chat(L, "<span class='warning'>You suddenly feel gorged!</span>")
+
+/obj/item/slimecross/destabilized/bluespace
+	colour = "bluespace"
+	effect_desc = "Teleports the hit creature next to the thrower."
+
+/obj/item/slimecross/destabilized/bluespace/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		do_teleport(A, get_turf(thrower), 1)
+	if(isobj(A))
+		var/obj/O = A
+		if(!O.anchored)
+			do_teleport(O, get_turf(thrower), 1)
+
+/obj/item/slimecross/destabilized/sepia
+	colour = "sepia"
+	effect_desc = "Briefly stops time for the hit creature, or temporaily stops time on the hit tile."
+
+/obj/item/slimecross/destabilized/sepia/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/datum/proximity_monitor/advanced/timestop/TS = new()
+		TS.host = A
+		TS.freeze_atom(A)
+		to_chat(A, "<span class='warning'>You suddenly feel every molecule in your body grind to a halt!</span>")
+		QDEL_IN(TS, 70)
+	else
+		new /obj/effect/timestop(get_turf(A), null, 50)
+
+/obj/item/slimecross/destabilized/cerulean
+	colour = "cerulean"
+	effect_desc = "Creates a weak, hostile, copy of the hit creature or the thrower if nothing is hit."
+
+/obj/item/slimecross/destabilized/cerulean/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(ismovableatom(A))
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), A, thrower)
+	else
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), thrower, thrower)
+
+/obj/item/slimecross/destabilized/pyrite
+	colour = "pyrite"
+	effect_desc = "Randomly colors everthing on the tile it hits."
+
+/obj/item/slimecross/destabilized/pyrite/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/turf/T = get_turf(A)
+	T.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+	for(var/atom/atoms in T)
+		atoms.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+
+/obj/item/slimecross/destabilized/red
```

Comment:
This one could be stronger, from what I've tested it only results in 5% total blood loss.